BranchSDK/src/BranchIO/Event/SessionEvent.h

```diff
@@ -32,10 +31,9 @@ class BRANCHIO_DLL_EXPORT SessionOpenEvent : public SessionEvent {
  public:
      /**
       * Constructor.
-      * @param jsonPtr pre-filled JSONObject to seed this class.
       */
```

Comment:
Wouldn't this be the payload (i.e. "intent data") that the app passes to the backend, which may or may not affect he response from the backend? Seems like a needed piece even if it's not _currently_ used. The same applies to all events where you removed the payload.

Reply:
I removed these because at this point they are untested and I don't think that they are needed.   It was originally put there because of the way the base class can take a pre-filled in JSONObject and my original expectation was to carry that forward...  but these specialized events shouldn't need this.

Reply:
kk, we can add them back in when they/if they become needed